doc/nrf/ug_multi_image.rst

```diff
@@ -24,12 +42,148 @@ MCUboot bootloader
+Note that both the child image's application build scripts and the core build scripts are executed.
+The core build scripts might use a different configuration and possibly different DeviceTree settings.
+
+Adding configuration options
+============================
+
+When enabling a child image, you select the build strategy, thus how the image is included.
+The three options are:
+
+* Build the child image from source along with the parent image - *IMAGENAME*\_BUILD_STRATEGY_FROM_SOURCE
+* Merge the specified HEX file of the child image with the parent image - *IMAGENAME*\_BUILD_STRATEGY_USE_HEX_FILE, and *IMAGENAME*\_HEX_FILE to specify the HEX file
+* Ignore the child image when building and build only the parent image - *IMAGENAME*\_BUILD_STRATEGY_SKIP_BUILD
+
+You must add these four configuration options to the Kconfig file for your child image, replacing *IMAGENAME* with the (uppercase) name of your child image (as specified in ``add_child_image``).
+
+The following example shows the configuration options for MCUboot:
+
+.. code-block:: Kconfig
+
+   choice
+  	prompt "MCUboot build strategy"
+  	default MCUBOOT_BUILD_STRATEGY_FROM_SOURCE
+
+   config MCUBOOT_BUILD_STRATEGY_USE_HEX_FILE
+  	# Mandatory option when being built through 'add_child_image'
+  	bool "Use HEX file instead of building MCUboot"
+
+   if MCUBOOT_BUILD_STRATEGY_USE_HEX_FILE
+
+   config MCUBOOT_HEX_FILE
+  	# Mandatory option when being built through 'add_child_image'
+  	string "MCUboot HEX file"
+
+   endif # MCUBOOT_USE_HEX_FILE
+
+   config MCUBOOT_BUILD_STRATEGY_SKIP_BUILD
+  	# Mandatory option when being built through 'add_child_image'
+  	bool "Skip building MCUboot"
+
+   config MCUBOOT_BUILD_STRATEGY_FROM_SOURCE
+  	# Mandatory option when being built through 'add_child_image'
+  	bool "Build from source"
+
+   endchoice
+
+
+Image-specific variables
+========================
+
+The child image and parent image are executed in different CMake processes and thus have different namespaces.
+Variables in the parent image are not propagated to the child image, with the following exceptions:
+
+* Any variable named *IMAGENAME*\_FOO in a parent image is propagated to the child image named *IMAGENAME* as FOO.
+* Variables that are in the list ``SHARED_MULTI_IMAGE_VARIABLES`` are propagated to all child images.
+
+With these two mechanisms, it is possible to set variables in child images from either parent images or the command line, and it is possible to set variables globally across all images.
+For example, to change the ``CONF_FILE`` variable for the MCUboot image and the parent image, specify the CMake command as follows::
+
+   cmake -Dmcuboot_CONF_FILE=prj_a.conf -DCONF_FILE=app_prj.conf
+
+The command line that is used to create the child images can be extended by adding flags to the CMake variable ``EXTRA_MULTI_IMAGE_CMAKE_ARGS``.
```

Comment:
```suggestion
You can extend the build command that is used to create the child images by adding flags to the CMake variable ``EXTRA_MULTI_IMAGE_CMAKE_ARGS``.
```

Reply:
No, invoking CMake is not building, it is configuring.

It's the ninja command that is the build command.